### PR TITLE
Chore: merge main@cd20834

### PR DESCRIPTION
**Pull request by julien-deramond**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,11 +190,16 @@ Our default branch is for development of our Boosted 5 release. Head to the [`v4
 ## Thanks
 
 <a href="https://www.browserstack.com/">
-  <img src="https://live.browserstack.com/images/opensource/browserstack-logo.svg" alt="BrowserStack Logo" width="192" height="42">
+  <img src="https://live.browserstack.com/images/opensource/browserstack-logo.svg" alt="BrowserStack" width="192" height="42">
 </a>
 
 Thanks to [BrowserStack](https://www.browserstack.com/) for providing the infrastructure that allows us to test in real browsers!
 
+<a href="https://www.netlify.com/">
```
Comment (louismaximepiton) [inline README.md:198]: I think there's a dupe here, maybe miw them all together to fit to Bootstrap best ?

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
A little dupe in the README.md.

File: README.md (hunk 1 above)
Comment (julien-deramond) [reply README.md:198]: Oh yes! Thanks! Fixed in a7152722c5ec8ee0e7180cfc650aea78df2b6176

**Review by louismaximepiton — APPROVED** [review]
Sounds good to me.

**Pull request merged**